### PR TITLE
Update the creating category rules page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request updates the creating category rules page

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/catalog/category-product-rules.html

### CODE REVIEW

File: src/catalog/category-product-rules.md [hunk 1]
```diff
@@ -38,14 +38,22 @@ Use caution when applying a category product rule, because any products that do
 
 1. Click <span class="btn">Add Condition</span> and do the following:
```
Comment (leslietilling) [inline src/catalog/category-product-rules.md:39]: If you convert bullet items to steps at same level, you have to remove this part

```suggestion
1. Click <span class="btn">Add Condition</span>.
```

File: src/catalog/category-product-rules.md [hunk 2]
```diff
@@ -55,12 +63,15 @@ Use caution when applying a category product rule, because any products that do
 
     - Set **Logic** to `AND`.
 
+      |`AND`|The rule applies to products, where all the conditions coincide.|
```
Comment (leslietilling) [inline src/catalog/category-product-rules.md:66]:
```suggestion
      |`AND`|The rule applies to products, where all the conditions are met.|
```

File: src/catalog/category-product-rules.md [hunk 3]
```diff
@@ -55,12 +63,15 @@ Use caution when applying a category product rule, because any products that do
...
+      |`OR`|The rule applies to products, where at least one condition coincides.|
```
Comment (leslietilling) [inline src/catalog/category-product-rules.md:67]:
```suggestion
      |`OR`|The rule applies to products, where at least one condition is met.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

**Comment by ZhovnirMaria** [comment]
Thanks, @leslietilling 😊
Done✅

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Pull request merged**